lib/ansible/runner.rb

```diff
@@ -210,9 +212,9 @@ def run_via_cli(hosts, credentials, env_vars, extra_vars, tags: nil, ansible_run
 
         create_hosts_file(base_dir, hosts)
         create_extra_vars_file(base_dir, extra_vars_hash)
-        create_cmdline_file(base_dir, command_line_hash)
 
         params = runner_params(base_dir, ansible_runner_method, playbook_or_role_args, verbosity)
+        create_cmdline_file(base_dir, params)
```

Comment:
I don't understand this change and how it applies to debugging?  command_line_hash is not part of the runner_params call, so is that just lost/not used?